### ISSUE TITLE
Color-Issue in upcoming cards ...

### ISSUE THREAD

**Issue by MWL1701** [nextcloud/deck#4386]
Hello,
I would like to report or ask, if there might be a solution for this issue ...
Is it possible to change the colors of the background and text for the times of deck-cards like in the following screenshot:
Here it is a orange/amber-background with a light-gray text-color which is very difficult to read because of the contrast between the colors:
![image](https://user-images.githubusercontent.com/90970568/212636213-46f99747-9898-4ce7-aa85-ca980c84a7cb.png)

I think the bg-color could stay as it is, but the text-color should be black. Then it is much more readable, in my opinion.

**Client details:**
 - OS: All OS (Windows/iOS)
 - Browser: chrome / iOS-App
 - Version: current version

**Operating system**: ubuntu server 22.04

**Web server:** Apache2

**Database:** mariadb

**PHP version:** 8.1

**Nextcloud version:** (see Nextcloud admin page) 25.0.2
